Marlin/configuration_store.cpp

```diff
@@ -787,8 +787,6 @@ void Config_ResetDefault() {
   #if ENABLED(PIDTEMP)
     #if ENABLED(PID_PARAMS_PER_HOTEND) && HOTENDS > 1
       HOTEND_LOOP()
```

Comment:
Will `e` still be defined here?  Or do we need to remove the `HOTENDS > 1` conditional?